tsl/test/expected/plan_skip_scan_compressed-12.out

```diff
@@ -0,0 +1,2138 @@
+CREATE TABLE sensor_data(
+time timestamptz not null,
+sensor_id integer not null,
+cpu double precision null,
+temperature double precision null );
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-01-01 00:00:00-00', 1, 0.5, 80);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-01-01 00:00:00-01', 2, 0.6, 70);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-02-01 00:00:00-00', 3, 0.9, 170);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-02-01 00:00:00-04', 4, 0.8, 140);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-03-01 00:00:00-00', 5, 0.9, 150);
+CREATE TABLE sensor_data_ht(
+time timestamptz not null,
+sensor_id integer not null,
+cpu double precision null,
+temperature double precision null );
+SELECT FROM CREATE_hypertable('sensor_data_ht','time', chunk_time_interval=> INTERVAL '30 day');
+--
+(1 row)
+
+INSERT INTO sensor_data_ht
+SELECT * FROM sensor_data;
+SELECT show_chunks('sensor_data_ht');
+              show_chunks               
+----------------------------------------
+ _timescaledb_internal._hyper_1_1_chunk
+ _timescaledb_internal._hyper_1_2_chunk
+ _timescaledb_internal._hyper_1_3_chunk
+(3 rows)
+
+CREATE INDEX ON sensor_data_ht (sensor_id);
+CREATE INDEX ON sensor_data (sensor_id);
+ALTER TABLE sensor_data_ht SET(timescaledb.compress, timescaledb.compress_orderby='time', timescaledb.compress_segmentby='sensor_id');
+VACUUM sensor_data;
+VACUUM sensor_data_ht;
+SET enable_seqscan = FALSE;
+:PREFIX SELECT DISTINCT ON (sensor_id)* FROM sensor_data_ht;
+                                                                  QUERY PLAN                                                                  
+----------------------------------------------------------------------------------------------------------------------------------------------
+ Unique (actual rows=5 loops=1)
+   Output: _hyper_1_1_chunk."time", _hyper_1_1_chunk.sensor_id, _hyper_1_1_chunk.cpu, _hyper_1_1_chunk.temperature
+   ->  Merge Append (actual rows=5 loops=1)
+         Sort Key: _hyper_1_1_chunk.sensor_id
+         ->  Index Scan using _hyper_1_1_chunk_sensor_data_ht_sensor_id_idx on _timescaledb_internal._hyper_1_1_chunk (actual rows=2 loops=1)
+               Output: _hyper_1_1_chunk."time", _hyper_1_1_chunk.sensor_id, _hyper_1_1_chunk.cpu, _hyper_1_1_chunk.temperature
+         ->  Index Scan using _hyper_1_2_chunk_sensor_data_ht_sensor_id_idx on _timescaledb_internal._hyper_1_2_chunk (actual rows=2 loops=1)
+               Output: _hyper_1_2_chunk."time", _hyper_1_2_chunk.sensor_id, _hyper_1_2_chunk.cpu, _hyper_1_2_chunk.temperature
+         ->  Index Scan using _hyper_1_3_chunk_sensor_data_ht_sensor_id_idx on _timescaledb_internal._hyper_1_3_chunk (actual rows=1 loops=1)
+               Output: _hyper_1_3_chunk."time", _hyper_1_3_chunk.sensor_id, _hyper_1_3_chunk.cpu, _hyper_1_3_chunk.temperature
```

Comment:
Why is this not using skipscan?

Reply:
Table is not compressed at this point. Compression happens right after this vanilla call.